### ISSUE TITLE
Invalid crop tier provided: mysticalagradditions:6

### ISSUE THREAD

**Issue by hyygybs** [BlakeBr0/MysticalCustomization#32]
**Describe the bug**
tier 6 crop doesn't work.
but mysticalagradditions:6 is in the list from /mysticalcustomization tiers

**To Reproduce**
```
{
    "name": "Neutronium",
    "type": "mysticalagriculture:resource",
    "tier": "mysticalagradditions:6",
    "ingredient": {
      "tag": "forge:ingots/neutronium"
    },
    "color": "0E0E0E",
    "textures": {
      "flower": "kubejs:block/neutronium_crop",
      "essence": "kubejs:item/neutronium_essence",
      "seeds": "kubejs:item/neutronium_seeds"
    },
    "crux": "kubejs:neutronium_crux"
  }

```
**Expected behavior**

**Screenshots / Scripts / Logs**
```
[12:35:15] [Render thread/ERROR]: An error occurred while creating crop with id mysticalcustomization:neutronium
com.google.gson.JsonSyntaxException: Invalid crop tier provided: mysticalagradditions:6
        at com.blakebr0.mysticalcustomization.create.CropCreator.create(CropCreator.java:26) ~[MysticalCustomization-1.18.2-3.1.2.jar%23308!/:3.1.2]
        at com.blakebr0.mysticalcustomization.loader.CropLoader.onRegisterCrops(CropLoader.java:57) ~[MysticalCustomization-1.18.2-3.1.2.jar%23308!/:3.1.2]
        at com.blakebr0.mysticalcustomization.lib.ModCorePlugin.onRegisterCrops(ModCorePlugin.java:20) ~[MysticalCustomization-1.18.2-3.1.2.jar%23308!/:3.1.2]
        at com.blakebr0.mysticalagriculture.registry.CropRegistry.lambda$onRegisterBlocks$2(CropRegistry.java:112) ~[MysticalAgriculture-1.18.2-5.1.4.jar%23307!/:5.1.4]
        at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
        at com.blakebr0.mysticalagriculture.registry.PluginRegistry.forEach(PluginRegistry.java:50) ~[MysticalAgriculture-1.18.2-5.1.4.jar%23307!/:5.1.4]
        at com.blakebr0.mysticalagriculture.registry.CropRegistry.onRegisterBlocks(CropRegistry.java:109) ~[MysticalAgriculture-1.18.2-5.1.4.jar%23307!/:5.1.4]
        at com.blakebr0.mysticalagriculture.init.ModBlocks.onRegisterBlocks(ModBlocks.java:131) ~[MysticalAgriculture-1.18.2-5.1.4.jar%23307!/:5.1.4]
        at net.minecraftforge.eventbus.ASMEventHandler_551_ModBlocks_onRegisterBlocks_Register.invoke(.dynamic) ~[?:?]
        at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-5.0.3.jar%232!/:?]
        at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-5.0.3.jar%232!/:?]
        at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-5.0.3.jar%232!/:?]
        at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:106) ~[javafmllanguage-1.18.2-40.2.1.jar%23451!/:?]
        at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:107) ~[fmlcore-1.18.2-40.2.1.jar%23450!/:?]
        at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?]
        at net.minecraftforge.fml.ModWorkManager$SyncExecutor.driveOne(ModWorkManager.java:42) ~[fmlcore-1.18.2-40.2.1.jar%23450!/:?]
        at net.minecraftforge.fml.ModWorkManager$DrivenExecutor.drive(ModWorkManager.java:26) ~[fmlcore-1.18.2-40.2.1.jar%23450!/:?]
        at net.minecraftforge.fml.ModLoader.waitForTransition(ModLoader.java:202) ~[fmlcore-1.18.2-40.2.1.jar%23450!/:?]
        at net.minecraftforge.fml.ModLoader.lambda$dispatchAndHandleError$21(ModLoader.java:187) ~[fmlcore-1.18.2-40.2.1.jar%23450!/:?]
        at java.util.Optional.ifPresent(Optional.java:178) ~[?:?]
        at net.minecraftforge.fml.ModLoader.dispatchAndHandleError(ModLoader.java:187) ~[fmlcore-1.18.2-40.2.1.jar%23450!/:?]
        at net.minecraftforge.fml.ModLoader.lambda$gatherAndInitializeMods$11(ModLoader.java:164) ~[fmlcore-1.18.2-40.2.1.jar%23450!/:?]
        at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?]
        at net.minecraftforge.fml.ModLoader.gatherAndInitializeMods(ModLoader.java:164) ~[fmlcore-1.18.2-40.2.1.jar%23450!/:?]
        at net.minecraftforge.client.loading.ClientModLoader.lambda$begin$1(ClientModLoader.java:92) ~[forge-1.18.2-40.2.1-universal.jar%23454!/:?]
        at net.minecraftforge.client.loading.ClientModLoader.lambda$createRunnableWithCatch$4(ClientModLoader.java:112) ~[forge-1.18.2-40.2.1-universal.jar%23454!/:?]
        at net.minecraftforge.client.loading.ClientModLoader.begin(ClientModLoader.java:92) ~[forge-1.18.2-40.2.1-universal.jar%23454!/:?]
        at net.minecraft.client.Minecraft.<init>(Minecraft.java:459) ~[client-1.18.2-20220404.173914-srg.jar%23449!/:?]
        at net.minecraft.client.main.Main.main(Main.java:169) ~[client-1.18.2-20220404.173914-srg.jar%23449!/:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31) ~[fmlloader-1.18.2-40.2.1.jar%2317!/:?]
        at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%235!/:?]
        at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%235!/:?]
        at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%235!/:?]
        at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%235!/:?]
        at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%235!/:?]
        at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%235!/:?]
        at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%235!/:?]
        at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?]
```

**Versions (please complete the following information):**
 - Minecraft:1.18.2
 - Forge:40.21
 - Cucumber:5.13
 - Mystical Agriculture:5.14
 - Mystical Agradditions:5.1.3
 - Mystical Customization:3.12

**Comment by hyygybs** [comment]
when i set it "mysticalagricultures:5"
that still have error "Invalid crop tier provided: mysticalagricultures:5"

**Comment by BlakeBr0** [comment]
Unfortunately the fix for this didn't make it into the 1.18 version. I think as a work around you can add your own 'copy' of the crop tier:

```
{
  "name": "6",
  "value": 6,
  "color": "40005E",
  "farmland": "mysticalagradditions:insanium_farmland",
  "essence": "mysticalagradditions:insanium_essence",
  "fertilizable": false,
  "secondarySeedDrop": false
}
```

Docs: https://blakesmods.com/docs/mysticalcustomization/adding-tiers
